l10n_it_fatturapa/models/account.py

```diff
@@ -327,6 +327,7 @@ class account_invoice(orm.Model):
    #_position = ['2.1', '2.2', '2.3', '2.4', '2.5']
     _inherit = "account.invoice"
     _columns = {
+        'protocol_number':  fields.char('Protocol Number', size=64),
```

Comment:
Questo dovrebbe essere aggiunto nel modulo del protocollo. O in un modulo di collegamento fra i moduli protocollo e account